set.mm

```diff
@@ -558947,6 +559324,18 @@ We use this concept to define functions ( ~~? df-funsALTV , ~~?
     ( ccoss wsymrel ccnv wss wrel wa symrelcoss2 dfsymrel2 mpbir ) ABZCKDKEKFGA
     HKIJ $.
 
+  $( The identity relation is symmetric.  (Contributed by AV, 19-Jun-2022.) $)
+  idsym $p |- SymRel _I $=
+    ( cid wsymrel ccnv wceq wrel cnvi reli dfsymrel4 mpbir2an ) ABACADAEFGAHI
+    $.
+
+  $( The membership (epsilon) relation is not symmetric.  (Contributed by AV,
+     18-Jun-2022.) $)
+  epnsym $p |- -. SymRel _E $=
+    ( cep wsymrel ccnv wceq wrel wa epasym neii intnanr dfsymrel4 mtbir ) ABACZ
+    ADZAEZFMNLAGHIAJK $.
```

Comment:
@mazsa I hope you don't mind that I have put these two theorems into your mathbox.

Reply:
@avekens of course not: feel free to use my Mathbox any time. (I'll continue to submit my theorems in July. This year:)